### PR TITLE
Update characterswitch.js

### PR DESCRIPTION
**Pull request by Safe870**
Some commas.

### CODE REVIEW

File: loc/ui/characterswitch.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default {
     ja: 'ログアウト',
     se: 'Logga ut',
     ph: 'Mag-log Out',
-    nl: 'Uitloggen'
+    nl: 'Uitloggen',
```
Comment (KaiserCheezel) [inline loc/ui/characterswitch.js]:
```suggestion
    nl: 'Uitloggen'
```

File: loc/ui/characterswitch.js [hunk 2]
```diff
@@ -31,6 +31,6 @@ export default {
     ja: 'ホームページ',
     se: 'Hemsida',
     ph: 'Home page',
-    nl: 'Thuis'
+    nl: 'Thuis',
```
Comment (KaiserCheezel) [inline loc/ui/characterswitch.js]:
```suggestion
    nl: 'Thuis'
```

**Comment by zbot473** [comment]
This changes nothing

**Comment by KaiserCheezel** [comment]
May as well just close this @drifted-mdl

**Pull request closed without merging**